src/lang/en.json

```
@@ -90,39 +90,9 @@
     "Ignore Sec-WebSocket-Accept header": "Ignore {0} header",
-    "wsSubprotocolDescription": "For more information on subprotocols, please consult the {documentation}",
-    "WebSocket Application Messaging Protocol": "WAMP (The WebSocket Application Messaging Protocol)",
-    "Session Initiation Protocol": "WebSocket Transport for SIP (Session Initiation Protocol)",
-    "Subprotocol": "Subprotocol",
-    "Network API for Notification Channel": "OMA RESTful Network API for Notification Channel",
-    "Web Process Control Protocol": "Web Process Control Protocol (WPCP)",
-    "Advanced Message Queuing Protocol": "Advanced Message Queuing Protocol (AMQP) 1.0+",
-    "jsflow": "jsFlow pubsub/queue Protocol",
-    "Reverse Web Process Control": "Reverse Web Process Control Protocol (RWPCP)",
-    "Extensible Messaging and Presence Protocol": "WebSocket Transport for the Extensible Messaging and Presence Protocol (XMPP)",
-    "Smart Home IP": "SHIP - Smart Home IP",
-    "Miele Cloud Connect Protocol": "Miele Cloud Connect Protocol",
-    "Push Channel Protocol": "Push Channel Protocol",
-    "Message Session Relay Protocol": "WebSocket Transport for MSRP (Message Session Relay Protocol)",
-    "Binary Floor Control Protocol": "WebSocket Transport for BFCP (Binary Floor Control Protocol)",
-    "Softvelum Low Delay Protocol": "Softvelum Low Delay Protocol",
-    "OPC UA Connection Protocol": "OPC UA Connection Protocol",
-    "OPC UA JSON Encoding": "OPC UA JSON Encoding",
-    "Swindon Web Server Protocol": "Swindon Web Server Protocol (JSON encoding)",
-    "Broadband Forum User Services Platform": "USP (Broadband Forum User Services Platform)",
-    "Constrained Application Protocol": "Constrained Application Protocol (CoAP)",
-    "Softvelum WebSocket signaling protocol": "Softvelum WebSocket Signaling Protocol",
-    "Cobra Real Time Messaging Protocol": "Cobra Real Time Messaging Protocol",
-    "Declarative Resource Protocol": "Declarative Resource Protocol",
-    "BACnet Secure Connect Hub Connection": "BACnet Secure Connect Hub Connection",
-    "BACnet Secure Connect Direct Connection": "BACnet Secure Connect Direct Connection",
-    "WebSocket Transport for JMAP": "WebSocket Transport for JMAP (JSON Meta Application Protocol)",
-    "ITU-T T.140 Real-Time Text": "ITU-T T.140 Real-Time Text",
-    "Done.best IoT Protocol": "Done.best IoT Protocol",
-    "Collection Update": "The Collection Update Websocket Subprotocol",
-    "Text IRC Protocol": "Text IRC Protocol",
-    "Binary IRC Protocol": "Binary IRC Protocol",
-    "Penguin Statistics Live Protocol v3": "Penguin Statistics Live Protocol v3 (Protobuf encoding)",
+    "wsSubprotocolDescription": "Enter a comma delimited list of subprotocols. For more information on subprotocols, please consult the {documentation}",
+    "wsCodeDescription": "For more information on status codes, please consult {documentation}",
```

makes it simpler on our translators (+removes the mistake that I did in the last review)

```suggestion
    "wsCodeDescription": "For more information on status codes, please consult {rfc6455}",
```